---
bool success
string message
poi_state_server/POIInfo[] pois

